docs/development.rst

```diff
@@ -20,23 +67,32 @@ Running Tests
 -------------
 
 Tests in pySMT are developed using python's built-in testing framework
-unittest. Each TestCase is stored into a separate file, and it should
-be possible to launch it by calling the file directly, e.g.:
+```unittest```.  Each ```TestCase``` is stored into a separate file,
```

Comment:
I think it is not necessary to escape unittest and TestCase. For emphasis, we can use italics or bold.